src/scenes/AcceleratorRouter/Cohorts/CohortEditView.tsx

```diff
@@ -72,19 +79,14 @@ export default function CohortEditView(): JSX.Element {
   }
 
   return (
-    <PageWithModuleTimeline
```

Comment:
We can check with designs but we likely do want to keep module stepper. I think in older mocks they have them.

Reply:
@chudo76 do we want to keep module stepper on Cohort View and Edit View?

Reply:
Michael probably designed a lot of these screens. I see that we have the stepper in Edit for others (Participant, Project) so we should have it for Cohort, although it feels a bit weird for all of them, and I don't exactly know how useful it is.

Reply:
on my second review, this seems redundant given the module table on the page. happy to defer to Michael to make the final call!!

Reply:
I also just went back to look at the designs ([Cohort Edit](https://www.figma.com/design/Ec5QnRsnzqEKeifzwPKR3k/Accelerator-Console?node-id=1654-24923&t=oxxdxOmy1AZzHSAN-4), [Project Edit](https://www.figma.com/design/Ec5QnRsnzqEKeifzwPKR3k/Accelerator-Console?node-id=1725-46570&t=oxxdxOmy1AZzHSAN-4)) and it looks like Michael didn't have the stepper for Edit mode. If it's not too much trouble, can we remove the stepper for the Edit modes for all of them?

Reply:
Oh I think I focused on the question about the _Edit_ page and missed the question about the _View_ page for Cohort Details.

I tend to agree and with your second review @tommylau523 , and for Cohort Details view, we don't need the Stepper since we will have the Modules table now. (cc: @constanzauanini)

Reply:
Thank you both!